private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -535,12 +1026,11 @@ export class TemplateApi {
                 localVarRequestOptions.form = localVarFormParams;
             }
         }
-        return new Promise<{ response: http.IncomingMessage; body: TemplateList;  }>((resolve, reject) => {
+        return new Promise<{ response: http.IncomingMessage; body?: any;  }>((resolve, reject) => {
```

Comment:
intentional change?

Reply:
the api.ts file was generated, but I'll make a note to look through the generated file and fix it up once the endpoint definitions are set!